### ISSUE TITLE
Doesn't build on nightly after rustc_front refactor

### ISSUE THREAD

**Issue by gyscos** [mcoffin/syntaxext_lint#6]
https://github.com/rust-lang/rust/pull/32570

```
syntaxext_lint-0.1.4/src/lib.rs:8:1: 8:26 error: can't find crate for `rustc_front` [E0463]
syntaxext_lint-0.1.4/src/lib.rs:8 extern crate rustc_front;
                                  ^~~~~~~~~~~~~~~~~~~~~~~~~
error: aborting due to previous error
```

**Comment by gyscos** [comment]
Fix is in #7

**Comment by mcoffin** [comment]
Fixed by 574b65a7807749353af1e75dac5b0612120fd0f6
